tests/integration/test_vulnerability_detector/test_general_settings/test_run_on_start.py

```diff
@@ -1,20 +1,30 @@
 '''
-copyright: Copyright (C) 2015-2021, Wazuh Inc.
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
 
            Created by Wazuh, Inc. <info@wazuh.com>.
 
            This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 
 type: integration
 
+<<<<<<< HEAD
```

Comment:
Conflicts have not been properly resolved

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/2649/commits/71e4587f39df3f93b1e16f8387b2c14fb5bc217b